src/test/java/HomeWork22.java

```diff
@@ -0,0 +1,52 @@
+import org.openqa.selenium.*;
+import java.time.Duration;
+
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.LoginPage;
+
+import static org.openqa.selenium.By.cssSelector;
+
+public class HomeWork22 extends BaseTest2{
+    @Test
+    public void renamePlaylist() {
+        LoginPage loginPage = new LoginPage(driver);
+        String expectedPlayListMsg = "Updated playlist \"Test1.\"";
+
+        loginPage.loginMainPage();
+        selectPlayList();
+
+        Assert.assertEquals(getUpdatedPlaylistMsg(), expectedPlayListMsg);
+
+    }
```

Comment:
Use in test methods, the following logical split:   GIVEN – WHEN - THEN
    
    
![Clean_Test](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/0faf6aa8-364d-46bf-97e5-e3062d239eac)

Reply:
is renamePlaylist correctly approached since there is a 'given - when - then'?

Reply:
Hello

Please, re-submit  for review